app/jobs/notify_webhook_job.rb

```diff
@@ -14,9 +14,13 @@ def perform(notification_id:)
     begin
       data = ActiveModelSerializers::Adapter.create(NotificationSerializer.new(notification)).to_json
       hmac = Encryptor.hmac(notification.subscription.secret, data)
```

Comment:
Maybe nice to assign `subscription = notification.subscription` here (or actually the line above) to simplify access throughout the method

Reply:
👍 yep that cleans it up a little, good suggestion